piptools/cache.py

```diff
@@ -51,6 +51,8 @@ class DependencyCache(object):
     def __init__(self, cache_dir=None):
         if cache_dir is None:
             cache_dir = CACHE_DIR
+        if not os.path.isdir(cache_dir):
+            os.makedirs(cache_dir)
```

Comment:
I didn't realize exist_ok existed. I was just mimicking these ones https://github.com/nvie/pip-tools/blob/master/piptools/repositories/pypi.py#L125-L128

I'll change it to your suggestion.